lib/numbers_in_words/number_parser.rb

```diff
@@ -1,4 +1,5 @@
 module NumbersInWords::NumberParser
+  SCALES_N = [100, 1000, 1000000, 1000000000, 1000000000000, 10**100]
```

Comment:
Sorry, that wasn't clear. We have some duplication of concepts here: https://github.com/markburns/numbers_in_words/blob/master/lib/numbers_in_words/english/constants.rb#L54

I've opened an issue for this: #7 

I suspect that this code isn't as comprehensive as it could be as a result, but I haven't tried to fully understand the years related stuff.

Anyway, this is merged in and we have an issue, no need to address it unless you feel like it.